src/audio/CMakeLists.txt

```diff
@@ -110,7 +110,12 @@ macro(check_optimization opt_name flag extra_define)
 	check_c_compiler_flag(${flag} compiles_flag_${opt_name})
 	if(compiles_flag_${opt_name})
 		list(APPEND available_optimizations ${opt_name})
-		set(${opt_name}_flags ${flag} ${extra_define} -ffast-math -ftree-vectorizer-verbose=0)
+		if (CMAKE_C_COMPILER_ID STREQUAL "GNU")
+			set(${opt_name}_flags ${flag} ${extra_define} -ffast-math -ftree-vectorizer-verbose=0)
+		else()
+			# clang doesn't -ftree-vectorizer-verbose
+			set(${opt_name}_flags ${flag} ${extra_define} -ffast-math)
+		endif()
```

Comment:
It is possible to `set(${opt_name}_flags ${flag} ${extra_define} -ffast-math)` unconditionally and then append `-ftree-vectorizer-verbose=0` if "GNU"? This would reduce duplication and get rid of the `else` clause.

Reply:
This entire PR was me googling through Cmakes docs so that might be possible

Reply:
Learning CMake is very much like doing taxes. Very painful at the beginning then it gets easier and easier over time. The pain never completely goes away.
Most people prefer to pay someone else to do it.